src/sentry/ingest/ingest_consumer.py

```diff
@@ -432,6 +433,13 @@ def get_ingest_consumer(
     The events should have already been processed (normalized... ) upstream (by Relay).
     """
     topic_names = {ConsumerType.get_topic_name(consumer_type) for consumer_type in consumer_types}
+
     return create_batching_kafka_consumer(
         topic_names=topic_names, worker=IngestConsumerWorker(executor), **options
     )
+
+
+def get_occurrences_ingest_consumer(
+    consumer_type,
+):
+    return create_ingest_occurences_consumer("occurrences")
```

Comment:
Wondering if this also belongs in `occurrence_consumer.py`

Reply:
yeah. will move and merge